identity/did/bootstrapper.go

```diff
@@ -66,99 +71,32 @@ func bindFactory(factoryAddress common.Address, client ethereum.Client) (*Factor
 	return NewFactoryContract(factoryAddress, client.GetEthClient())
 }
 
-func getFactoryAddress() common.Address {
-	return common.HexToAddress(smartContractAddresses.IdentityFactoryAddr)
-
-}
-
-func getAnchorAddress() common.Address {
-	return common.HexToAddress(smartContractAddresses.AnchorRepositoryAddr)
+func getFactoryAddress(cfg config.Configuration) common.Address {
+	return cfg.GetContractAddress(config.IdentityFactory)
 }
 
-// Note: this block will be removed after the identity migration is done
-// currently we are using two versions of centrifuge contracts to not break the compatiblitiy
-// ---------------------------------------------------------------------------------------------------------------------
-func migrateNewIdentityContracts() {
-	runNewSmartContractMigrations()
-	smartContractAddresses = GetSmartContractAddresses()
-
-}
-
-// RunNewSmartContractMigrations migrates smart contracts to localgeth
-// TODO: func will be removed after migration
-func runNewSmartContractMigrations() {
-
+// TODO: store identity bytecode in config and remove func
+func getIdentityByteCode() string {
```

Comment:
Where are we using this?

Reply:
We are using it for CreateIdentity to check if it was successful